src/local/BLELocalDevice.cpp

```diff
@@ -108,7 +108,7 @@ void BLELocalDevice::end()
 
   HCI.end();
 
-#if defined(ARDUINO_SAMD_MKRWIFI1010) || defined(ARDUINO_AVR_UNO_WIFI_REV2)
+#if defined(ARDUINO_SAMD_MKRWIFI1010) || defined(ARDUINO_AVR_UNO_WIFI_REV2) || defined(ARDUINO_SAMD_NANO_33_IOT)
```

Comment:
This doesn't work because RESETN pin is not inverted in NANO_33_IOT, so `digitalWrite(NINA_RESETN, HIGH);` enables the wireless module.

Reply:
confirming that changing that reset line to the folowing enables the module for BLE successfully  ````
 digitalWrite(NINA_RESETN, LOW);
  delay(100);
  digitalWrite(NINA_RESETN, HIGH);
  delay(750);
````

(borrowed from WiFiNINA lib), but there still seems to be a problem maintaining connection when a central connects. I suspect there's a reset somewhere that I am still missing.

Reply:
Update on this, it seems to be working now with those changes, though not using LightBlue on macOS. Works with BLE MIDI though, and with the various BLE Anrdoid apps.